RedirectApplication/RedirectApplication/Json/Json.cs

```diff
@@ -16,8 +16,13 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
+            if (reader.Path == "TargetUrl")
```

Comment:
Use markdown syntax to insert code examples into comments, i.e.:
```json
{
  "TargetUrl":"localhost",
  "Conditions": [ {"Name:":"ByBrowser","Browser":"Chrome","Url":"http://pikabu.ru/"} ]
}
```
will be generated from
```
```json
{
  "TargetUrl":"localhost",
  "Conditions": [ {"Name:":"ByBrowser","Browser":"Chrome","Url":"http://pikabu.ru/"} ]
}
``` 
```